### PR TITLE
add build-cop bz query to overview dashboard

### PR DESCRIPTION
**Pull request by bparees**
image

### CODE REVIEW

File: cmd/release-controller/http.go [hunk 1]
```diff
@@ -159,6 +159,8 @@ const releaseInfoPageHtml = `
 const releaseDashboardPageHtml = `
 <h1>Release Dashboard</h1>
 <hr>
+<h2><a class="text-dark" href=https://bugzilla.redhat.com/buglist.cgi?bug_status=NEW&bug_status=ASSIGNED&bug_status=POST&f1=cf_internal_whiteboard&f2=status_whiteboard&j_top=OR&known_name=BuildCop&list_id=10913331&o1=substring&o2=substring&query_format=advanced&v1=buildcop&v2=buildcop>Open Build Cop Bugs</a></h2>
```
Comment (smarterclayton) [inline cmd/release-controller/http.go]: I'd prefer just a normal text link.  This doesn't look like a link (generally H2 as link is frowned upon).

`<p><a ...></p>`

**Comment by bparees** [comment]
updated:
image

File: cmd/release-controller/http.go [hunk 2]
```diff
@@ -159,6 +159,8 @@ const releaseInfoPageHtml = `
 const releaseDashboardPageHtml = `
 <h1>Release Dashboard</h1>
 <hr>
+<p><a href=https://bugzilla.redhat.com/buglist.cgi?bug_status=NEW&bug_status=ASSIGNED&bug_status=POST&f1=cf_internal_whiteboard&f2=status_whiteboard&j_top=OR&known_name=BuildCop&list_id=10913331&o1=substring&o2=substring&query_format=advanced&v1=buildcop&v2=buildcop>Open Build Cop Bugs</a></p>
+<hr>
```
Comment (smarterclayton) [inline cmd/release-controller/http.go]: You shouldn't need an HR

Comment (smarterclayton) [reply cmd/release-controller/http.go]: In fact, remove the other HR too :)

HR are unnecessary in most contexts

**Pull request merged**